src/context.rs

```diff
@@ -488,11 +590,15 @@ impl BaseAudioContext {
             outputs: node.number_of_outputs() as usize,
             channel_config: node.channel_config_cloned(),
         };
-        self.inner.render_channel.send(message).unwrap();
+        self.inner
+            .render_channel
+            .send_timeout(message, Duration::from_millis(10))
```

Comment:
Why opt for timeout? If the requested latency `> 10millis`, this will fail for no good reason